### ISSUE TITLE
{gitreponame} does not display correctly

### ISSUE THREAD

**Issue by Zebiano** [iCrawl/discord-vscode#554]
I was just playing around with the settings inside the vscode extension, and noticed the following placeholder:
```
'{gitreponame}' will be replaced with the active Git repository name (from the git URL)
```
Unfortunately, it does not work as expected. Under the rich presence window, it's written `{gitreponame}` (literally).

Am I missing something? I feel like I'm not understanding the 'from git URL' correctly. What URL? Where is it stored?

**Comment by Zebiano** [comment]
Update: Now it's displaying `Unknown`. The only thing I've been doing is restarting VSCode several times to test stuff out.

**Comment by iCrawl** [comment]
The directory thats open in VSCode is a git repo yes? And git is installed?

**Comment by Zebiano** [comment]
Yes, the directory is a Git repo and Git is also installed: `git version 2.25.1.windows.1`.

Running `git config --get remote.origin.url` returned `https://github.com/Zebiano/Labeler.git`, which I'm guessing is what your extension should be/is searching for? Got is form [here](https://stackoverflow.com/a/4089452/8234174).
